src/static/css/index.css

```diff
@@ -335,6 +332,12 @@ p {
 }
 
 @media (max-width: 900px) and (min-width: 601px) {
+
+  header.alt-bg {
+    padding-bottom: 1800px;
```

Comment:
Previously, we had a single "magic number" e.g. `400px`  which we used to accommodate the slanted SVG background header. As we already chatted, that was implemented in #492 and helped us getting rid of some hardcoded/magic numbers.

It seems that now we have `900px`, `1650px` and `1800px`.

Reply:
Yes we do have more magic numbers in this one. Personally I think that's an acceptable trade off to fix the HTML.

As I mentioned in our chat, currently in Safari reader mode (on both desktop on MacOS and mobile in iOS) the home page does not render correctly and drops the intro text that I think is important for the home page.

I'm also using Safari reader mode as a bit of proxy for how other non-browsers will render the page.